DAO.sol

```diff
@@ -855,6 +878,7 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
     function minQuorum(uint _value) internal constant returns (uint _minQuorum) {
         // minimum of 20% and maximum of 53.33%
+        uint combinedTotalSupply = DAO(privateCreation).totalSupply() + totalSupply - tokensReceived;
```

Comment:
this will be done when it's combined with [this](https://github.com/slockit/DAO/pull/244) PR